Composer/packages/server/src/services/project.ts

```diff
@@ -278,4 +362,30 @@ export class BotProjectService {
       return '';
     }
   };
+
+  public static backupProject = async (project: BotProject): Promise<string> => {
```

Comment:
What's the difference between this and backupProject defined under controller

Reply:
This is another artifact from the larger feature branch. This same logic is used by the "Pull" feature which will be broken out into a separate PR following this one. I will delete this one for now until I open that PR.